catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/converter/JdbcColumnDefaultValueConverter.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.jdbc.converter;
+
+import static com.datastrato.gravitino.rel.Column.DEFAULT_VALUE_NOT_SET;
+
+import com.datastrato.gravitino.rel.expressions.Expression;
+import com.datastrato.gravitino.rel.expressions.FunctionExpression;
+import com.datastrato.gravitino.rel.expressions.literals.Literal;
+import com.datastrato.gravitino.rel.expressions.literals.Literals;
+import com.datastrato.gravitino.rel.types.Type;
+import java.time.format.DateTimeFormatter;
+
+public abstract class JdbcColumnDefaultValueConverter {
+
+  protected static final String CURRENT_TIMESTAMP = "CURRENT_TIMESTAMP";
+  protected static final String NULL = "NULL";
+  protected static final DateTimeFormatter DATE_TIME_FORMATTER =
+      DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+
+  public String fromGravitino(Expression defaultValue) {
+    if (DEFAULT_VALUE_NOT_SET.equals(defaultValue)) {
+      return null;
+    }
+
+    if (defaultValue instanceof FunctionExpression) {
+      FunctionExpression functionExpression = (FunctionExpression) defaultValue;
+      if (functionExpression.functionName().equalsIgnoreCase(CURRENT_TIMESTAMP)) {
+        return CURRENT_TIMESTAMP;
+      } else {
+        return String.format("(%s)", functionExpression);
+      }
+    }
+
+    if (defaultValue instanceof Literal) {
+      Literal<?> literal = (Literal<?>) defaultValue;
+      Type type = literal.dataType();
+      if (defaultValue.equals(Literals.NULL)) {
+        return NULL;
+      } else if (type instanceof Type.NumericType) {
+        return literal.value().toString();
+      } else {
+        return String.format("'%s'", literal.value());
+      }
+    }
+
+    throw new IllegalArgumentException("Not a supported column default value: " + defaultValue);
+  }
+
+  public abstract Expression toGravitino(
```

Comment:
suggest to provide a default implement for JDBC catalogs doesn't support default value

Reply:
This should be handled by catalog level capability framwork

Reply:
how capability framework handle this?

Reply:
I misunderstood. Default implement added